### ISSUE TITLE
🐛 Bug: .all-contributorsrc template is missing an emoji in its alt text

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/create-typescript-app#929]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

```md
<img alt="All Contributors: 30 🤝" src="https://img.shields.io/badge/all_contributors-30_🤝-21bb42.svg" />
```

### Actual

```md
<img alt="All Contributors: 30" src="https://img.shields.io/badge/all_contributors-30_🤝-21bb42.svg" />
```
